hadoop-examples/README.md

```diff
@@ -0,0 +1,94 @@
+<!--
+Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+-->
+
+# hadoop-connectors examples
+Examples of Hadoop Connectors for Pravega.
+
+Description
+-----------
+These examples give you some basic ideas how to use hadoop-connectors for pravega
+
+Build
+-------
+(Most of these steps will be removed when GAed)
+### build hadoop connectors
+```
+git clone --recurse-submodules https://github.com/pravega/hadoop-connectors.git
+cd hadoop-connectors
+gradle install
+
+# start pravega whose version matches hadoop-connectors
+cd pravega
+./gradlew startStandalone
+```
+
+## build hadoop connectors examples
+```
+open another terminal and goto ~/pravega-samples
+cd hadoop-examples
+gradle build
+```
+
+Run Examples on Local Machine
+---
+
+
+Hadoop (verified with Hadoop 2.8.3 on Ubuntu 16.04)
+```
+1. setup and start hdfs
+
+2. set env variables
+   export HDFS=hdfs://192.168.0.188:9000
```

Comment:
Maybe a `localhost` or `your_ip` would be clearer than an arbitrary IP address.

Reply:
accepted